package.json

```diff
@@ -69,13 +70,12 @@
     "@mojaloop/central-services-error-handling": "7.1.2",
     "@mojaloop/central-services-health": "7.0.0",
     "@mojaloop/central-services-metrics": "5.2.0",
-    "@mojaloop/central-services-shared": "6.4.1",
+    "@mojaloop/central-services-shared": "7.2.2-snapshot",
```

Comment:
Please first deal with this PR before we can merge this one as we need a public release --> https://github.com/mojaloop/central-services-shared/pull/57

Reply:
Awaiting approval on PR https://github.com/mojaloop/central-services-shared/pull/57

Reply:
mojaloop/central-services-shared#57 is now merged @rmothilal .. This conversation can be resolved.